CMake/AbseilHelpers.cmake

```diff
@@ -62,7 +62,114 @@ function(absl_library)
   endif()
 endfunction()
 
+#
+# CMake function to imitate Bazel's cc_library rule.
+#
+# Parameters:
+# NAME: name of target (see Note)
+# HDRS: List of public header files for the library
+# SRCS: List of source files for the library
+# DEPS: List of other libraries to be linked in to the binary targets
+# COPTS: List of private compile options
+# DEFINES: List of public defines
+# LINKOPTS: List of link options
+# VISIBILITY_PUBLIC: Add this so that this library will be exported under absl:: (see Note).
```

Comment:
`PUBLIC` is not reserved by CMake, but I just thought that `VISIBILITY_PUBLIC` is easier to understand for Abseil team since Bazel uses `//visibility:public` to mean that "targets outside Abseil can depend on it". I can change it to `PUBLIC` if you think that is better.

Reply:
Done.